src/main/java/cz/muni/fi/pa165/entity/User.java

```diff
@@ -0,0 +1,22 @@
+package cz.muni.fi.pa165.entity;
+
+/**
+ * Hello world!
+ *
+ */
+
+@Entity
+@Table(name="User")
+public class User
+{
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable=false,unique=true)
+    @Pattern(regexp=".+@.+\\....?")
```

Comment:
Neměli by nějakej email validátor? Jinak, ten regulár nebude fungovat pro delší top level tld, třeba .academy. A ty tři tečky za sebou by asi šly přepsat na .{3}, i když na počet zanků to delší nebude.

Reply:
Proslo by ti i " @ .xyz". Udelej tam aspon validaci na skupinu [a-zA-Z0-9_]+